### PR TITLE
Add CONTRIBUTING and CODE_OF_CONDUCT docs

### PR DESCRIPTION
**Pull request by saw-jan**
### Added
docs:
- CONTRIBUTING.md
- CODE_OF_CONDUCT.md

### Related Issues
part of #4

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
+# Contributor Covenant Code of Conduct
```
Comment (saw-jan) [inline CODE_OF_CONDUCT.md:1]: Code of COnduct is exactly taken from https://www.contributor-covenant.org/version/2/1/code_of_conduct/

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
+# How to Contribute
+
+We thank you for taking your time to contribute to our project. In this guide you will get an overview of the contribution workflow from opening an issue and creating a PR.
```
Comment (kiranparajuli589) [inline CONTRIBUTING.md:3]:
```suggestion
We thank you for taking the time to contribute to our project. In this guide, you will get an overview of the contribution workflow from opening an issue and creating a PR.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+The guidelines in this section can be used to create enhancement and new feature request tickets.
+
+Have cool enhancement or great feature requests? Then check the following guidelines to request one:
+
+1. Search in the existing [list](https://github.com/gherlint/gherlint/issues) and make sure your idea has not been reported already.
+2. Open a new issue with a relevant template.
+3. Add as much information as possible. Some valuable information might be:
+    - Descriptive title
+    - Clear description about the request
```
Comment (kiranparajuli589) [inline CONTRIBUTING.md:42]:
```suggestion
    - Clear description of the request
```

File: CODE_OF_CONDUCT.md [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+[INSERT CONTACT METHOD].
```
Comment (phil-davis) [inline CODE_OF_CONDUCT.md:63]: What to put here?

Comment (saw-jan) [reply CODE_OF_CONDUCT.md:63]: Ohh, yeah we need to put there some contact information (maybe email). What can we put there?

Comment (phil-davis) [reply CODE_OF_CONDUCT.md:63]: @individual-it what should we do about this?

Comment (individual-it) [reply CODE_OF_CONDUCT.md:63]: contact form on the website, the company leadership receives that email

Comment (grgprarup) [reply CODE_OF_CONDUCT.md:63]: IMO, might be a link to the report form or simply an email.

Comment (individual-it) [reply CODE_OF_CONDUCT.md:63]: publishing an email address would introduce even more spam

Comment (phil-davis) [reply CODE_OF_CONDUCT.md:63]: The company contact form is at https://www.jankaritech.com/#contact

**Review by grgprarup — COMMENTED** [review]
lgtm :+1:

**Review by phil-davis — APPROVED** [review]

**Pull request merged**